src/Entity/Udi.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+namespace App\Entity;
+
+use Doctrine\ORM\Mapping as ORM;
+use Symfony\Bridge\Doctrine\Validator\Constraints\UniqueEntity;
+
+/**
+ * Entity class to keep list of issued UDI's.
+ *
+ * @ORM\Entity(repositoryClass="App\Repository\UdiRepository")
+ *
+ * @UniqueEntity(
+ *     fields={"uniqueDataIdentifier"},
+ *     errorPath="uniqueDataIdentifier",
+ *     message="This UDI already exists"
+ * )
+ */
+class Udi extends Entity
```

Comment:
This really doesn't need to extend Entity. There is no need for modifier, on modified/created dates?
What do you guys think? @fightingtexasaggie @praneethpr

Reply:
As we talked about at stand up, I'm in agreement we don't need it. Would we have to add an ID to the class?

Reply:
No, not needed. UDI can be the identifier.

Reply:
I think its better to have ID to the class, because UDI isnt a good identifier

Reply:
UDI will be unique, it's just a list.